apps/www/tailwind.config.js

```diff
@@ -17,10 +17,23 @@ module.exports = config({
           '0%': { backgroundColor: 'rgba(63, 207, 142, 0.1)' },
           '100%': { backgroundColor: 'transparent' },
         },
+        fadeIn: {
```

Comment:
clash of keyframe names

https://github.com/supabase/supabase/blob/master/packages/config/ui.config.js/#L160-L167